internal/services/loganalytics/migration/workspace_v0_to_v1.go

```diff
@@ -51,6 +51,11 @@ func workspaceSchemaForV0AndV1() map[string]*pluginsdk.Schema {
 			ForceNew: true,
 		},
 
+		"force_cmk_for_query": {
```

Comment:
as this is a boolean
```suggestion
		"cmk_for_query_forced": {
```
or
```suggestion
		"cmk_for_query_enforced": {
```

Reply:
Updated